src/sp/it/pl/plugin/ScreenRotator.kt

```diff
@@ -49,6 +49,16 @@ class ScreenRotator: PluginBase("Screen Rotator", true) {
 
     fun openHelp() = programHelpFile.open()
 
+    private fun action(screen: Int?, keys: String) = Action(
```

Comment:
Please add a little documentation

Reply:
It is a private element and it is absolutely clear I think what it does? Documentation is important, but the thing with it is: less is more.

I'm not fun of private utility methods like this in the 1st place (and long pondered on whether to use it here, since I originally wrote the code in fact), but the benefits here speak for themselves. The documentation shouldnt be at all necessary.

Reply:
It is as clear as if the method body was inlined. I think the readability is a non issue here because it is a small, self contained class (almost a script file), where the details like the parameters above are unimportant implementation details.

For APIs it is incredibly important to remain as simple and helpful as possible, in private methods its good to, well, avoid them altogether and if necessary at least make them make sense - make them express single well defined (self--evident) concept and for convenience methods - avoid public ones like a plague and do private ones only if code reduction is substantial. As for documentation - public always, private never.

This is why I consider private convenience methods as target for 'readability optimization' - short names 1 letter parameters etc. They are effectively macros or local DSLs - very short and programmer should be only interested in the method body, the rest is noise.

Note that convenience method in this context is one that does not express a concept, like the action() method in our case. It serves just one case: making code shorter.

If you think the parameters here are too mysterious, Id rather remove the method altogether. However I think the parameters are self explanatory: Int screen is intuitively a screen index (with null signaling some sort of special case) and String keys is something that can be found throughout the entire application, due to lack of any class that effectively captures the concept. There is JavaFx' KeyCombination, but I felt like that is very cumbersome and dangerous to use. I'm thinking there could be a special class with a DSL builder where something like CTRL+ALT+X would be an operator plus() chain on type safe constants producing this custom type or something.